### DISCUSSION TITLE
[Project settings][Environments] - Env prefix of new environment

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#59] [Q&A] [answered]
Hi guys, a little question.
When creating a new `Environment` on a project on the Mia-Platform Console, must I include the `_` (underscore) inside the property `Environment variables prefix` or not?

For example, I am configuring the new `sandbox` environment on my project and I want to have environment variables like the following one:
> `SANDBOX_MONGODB_URL`

Thanks.

**Comment by fredmaggiowski** [comment]
Hi, I think that the `_` should be provided, so yes your prefix should be in the form `SANDBOX_`. @Danielecina may tell you more about it; also it should be specified in the docs, can we adjust it accordingly? https://docs.mia-platform.eu/docs/development_suite/set-up-infrastructure/runtime-environments

**Comment by davidebianchi** [comment, marked as answer]
At the moment, the default environment prefix at the project creation is without the underscore. So, it should not have the underscore also in the environment part.
BUT the environment prefix at the moment is not passed inside the trigger of the deploy, so it is not used. We should add it, to be used in the pipeline for example in Gitlab, when the user can prefix the environment variables. 
So, in conclusion, I think that the best way to set the envirnoment prefix would be with the `_` (otherwise, it would be needed in the pipeline add the underscore at the end of the prefix every time), but there is an inconsistency in the use of this data. So, to maintain consistency, at the moment you should use without the final `_`. We must work on it as soon as possible.

**Reply by frengooo** [reply]
Ok, so right now the user can't have another delimiter except the `_` between the prefix and the name of the variable?

>BUT the environment prefix at the moment is not passed inside the trigger of the deploy, so it is not used.

I didn't understood this point. The `Environment variables prefix` property is not used right now?
In the following situation, what is the prefix that will be used to correctly configure and deploy a project on the `sandbox` environment?
![Schermata 2022-10-21 alle 11 28 37](https://user-images.githubusercontent.com/28152586/197163267-c47cfde6-2a5f-4ec3-88a7-e1a377d5c23b.png)

**Reply by davidebianchi** [reply]
It should be set directly inside your pipeline flow
